tests/connectors/dim/margo_test.py

```diff
@@ -40,12 +41,17 @@ def test_connector_spawns_server() -> None:
     with mock.patch(
         'proxystore.connectors.dim.margo.wait_for_server',
         side_effect=ServerTimeoutError,
-    ), mock.patch(
-        'proxystore.connectors.dim.margo.Engine',
+    ), mock.patch.object(
```

Comment:
Let's use `mock.patch('proxystore.connectors.dim.margo.Engine')` here and remove the import of `MargoEngine`.